build/stage-build.yml

```diff
@@ -21,7 +21,7 @@
   inputs:
     command: 'custom'
     custom: 'workload'
-    arguments: 'install android ios macos maccatalyst'
+    arguments: 'install android macos maccatalyst wasm-tools-net6'
```

Comment:
Why do we need `wasm-tools-net6`?

Reply:
The new agents 3.230.0 that rolled out are requiring wasm-tools-net6, I'm investigating at the moment why it's needed, one option I'm considering is specifying the previous agent instead 2.213.2 of 3.230.0.
`##[error]C:\Program Files\dotnet\sdk\8.0.100\Sdks\Microsoft.NET.Sdk\targets\Microsoft.NET.Sdk.ImportWorkloads.targets(38,5): Error NETSDK1147: To build this project, the following workloads must be installed: wasm-tools-net6`
https://dev.azure.com/nventive/Open%20Source/_build/results?buildId=286259&view=logs&j=37cf3487-6013-5a1c-b240-ba6b5f2f06df&t=383b4e78-b707-5513-965a-0259dd1535cf&l=30

3.230.0 is causing issue even for a refactoring
https://github.com/nventive/AsyncWebView/pull/39

AsyncWebView Pipeline
https://dev.azure.com/nventive/Open%20Source/_build/results?buildId=286280&view=results

Reply:
Ok